src/components/AsideBlock/AsideBlock.scss

```diff
@@ -0,0 +1,40 @@
+@import '../../styles/variables';
+
+$asideblock-default-height: calc(100% - 62px);
```

Comment:
All variables that are reused somewhere else, must be moved to the 
`src/styles/_variables.scss` file. 

Here it is likely to be 62px.